invenio_github/api.py

```diff
@@ -147,8 +148,8 @@ def sync(self, sync_hooks=True):
                 'full_name': gh_repo.full_name,
                 'description': gh_repo.description,
             }
-            if sync_hooks:
-                self._sync_hook(gh_repo)
+
+        self._sync_hooks(active_repos.keys(), async=async_hooks)
```

Comment:
My question was more about keep the `sync_hooks` parameter so you can sync everything but hooks.